### ISSUE TITLE
Method isSelected(date) missing

### ISSUE THREAD

**Issue by federicoibba** [GramThanos/jsCalendar#3]
This calendar is good and provides all the functions someone can need, good job!
One enhancement that I could suggest is to implement a method to check if a certain date is already selected.

**Comment by GramThanos** [comment]
Good idea! I can add it.

And maybe some more isXY(..) methods would be nice, like:

```javascript
calendar.isVisible(date); // Returns true if date is displayed
calendar.isInMonth(date); // Returns true if date is in active month
```

BTW, I personaly prefer `isSelected` over `isSelect`.

**Comment by federicoibba** [comment]
Yes, that was the idea. I prefer isSelected too (isSelect is a bad grammatical error, sorry). 
I was searching for a good calendar to insert in a project and then I found yours. My need is to insert events into a calendar and doing something into it and a `isSelected `method would be great!

**Comment by GramThanos** [comment]
Implemented and are now live on developer version *v1.4-beta* for testing

`calendar.isSelected(date);` 8b47e12588c9db9868405a44ef023f5fd7a50c4c
`calendar.isVisible(date);` 733f819138d4c12a8df7a95392577e8bc02bc255
`calendar.isInMonth(date);` 7221cf353e8595a41e9b433eaf4f0e23bb12f1f2

Also, to be honest, when I implemented the first release, I just added the select methods as a simple minimal feature. So, some more features may be needed, like `calendar.getSelected();` (I will implement it).

**Comment by crist-y** [comment]
Hi mate, great job you're doing.
Is there a way to add a min date (ie: current date) and a maxdate (ie: current date = 1 year)?
Cheers!

**Comment by GramThanos** [comment]
Thanks. As this is a different issue, I opend one for it here #4.
Please express your thoughts over it there 😄

**Comment by GramThanos** [comment]
The method `getSelected` was implemented and is now live on developer version v1.4-beta for testing
`calendar.getSelected(options);` 3582a055ba969c4eda3ca0cfb7e77291a60b70b1

Examples : 
```javascript
myCalendar.getSelected();
// Returns [1483567200000, 1483394400000, 1483308000000, 1483221600000]

myCalendar.getSelected({sort:true});
// Returns [1483221600000, 1483308000000, 1483394400000, 1483567200000]

myCalendar.getSelected({sort:"desc",type:"DD-MM-YYYY"});
// Returns ["05-01-2017", "03-01-2017", "02-01-2017", "01-01-2017"]

myCalendar.getSelected({sort:"asc",type:"date"});
// Returns [Sun Jan 01 2017 00:00:00 GMT+0200 (GTB Standard Time), Mon Jan 02 2017 00:00:00 GMT+0200 (GTB Standard Time), Tue Jan 03 2017 00:00:00 GMT+0200 (GTB Standard Time), Thu Jan 05 2017 00:00:00 GMT+0200 (GTB Standard Time)]
```
